hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestPrefetch.java

```diff
@@ -96,6 +99,10 @@ public class TestPrefetch {
   private static final int DATA_BLOCK_SIZE = 2048;
   private static final int NUM_KV = 1000;
 
+  private long startTime;
+  private boolean measureTiming;
+
```

Comment:
Please don't add global variables that are only used by one specific test logic.